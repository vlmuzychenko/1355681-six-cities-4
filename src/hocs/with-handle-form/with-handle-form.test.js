import React from "react";
import renderer from "react-test-renderer";
import withHandleForm from "./with-handle-form.js";

const offerIdMock = 1;

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MockComponentWrapped = withHandleForm(MockComponent);

it(`withHandleForm is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped
      offerId={offerIdMock}
      onSubmit={() => {}}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
