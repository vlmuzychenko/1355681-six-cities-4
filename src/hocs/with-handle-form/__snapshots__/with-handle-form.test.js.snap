// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withHandleForm is rendered correctly 1`] = `<div />`;
